onnxruntime/core/graph/model.cc

```diff
@@ -284,6 +284,38 @@ Status Model::Load(std::unique_ptr<ModelProto> p_model_proto, std::shared_ptr<Mo
   return Status::OK();
 }
 
+template <typename T>
+static Status LoadModel(const T& file_path, ONNX_NAMESPACE::ModelProto& model_proto) {
+  int fd;
+  Status status = Env::Default().FileOpenRd(file_path, fd);
```

Comment:
> Env::Default() [](start = 17, length = 15)

Should we not use Env() that the user created?

Reply:
As I said, this is mostly the same code that was being used to create Model, but now adapted to just parse the ModelProto. Would we be better off just refactoring/refining all aspects of the Model class later ?

Reply:
Env() is needed here for a platform specific method

---
In reply to: [349250020](https://github.com/microsoft/onnxruntime/pull/2449#discussion_r349250020) [](ancestors = 349250020)